addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.js

```diff
@@ -90,13 +89,10 @@ export class SubtaskKanbanList extends Component {
 
     async _onSubtaskCreateNameChanged(name) {
         if (name.trim() === "") {
-            this.notification.add(
-                markup(`<ul><li>${escape(_t("Display Name"))}</li></ul>`),
```

Comment:
Note: it is a bit strange to build a list with always a single element, but I guess why not.